griptape/drivers/sql/amazon_redshift_sql_driver.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Optional
+import boto3
+from griptape.drivers import BaseSqlDriver
+from attr import define, field
+
+
+@define
+class AmazonRedshiftSqlDriver(BaseSqlDriver):
+    database: str = field(kw_only=True)
+    cluster_identifier: str = field(default=None, kw_only=True)
+    workgroup_name: str = field(default=None, kw_only=True)
+    db_user: str = field(default=None, kw_only=True)
+    secret_arn: str = field(default=None, kw_only=True)
+    region_name: str = field(default="us-east-1", kw_only=True)
+    wait_sec: float = field(default=0.3, kw_only=True)
+    client = field(default=None, kw_only=True)
+
+    def __attrs_post_init__(self):
+        if self.client == None:
+            session = boto3.Session(region_name=self.region_name)
+            self.client = session.client("redshift-data")
+
+    @staticmethod
```

Comment:
I know that it's technically the right way to do it but it results in too many `AmazonRedshiftSqlDriver`s all over the code. We could make it a class method instead:

```python
@classmethod
def _process_rows_from_records(cls, records) -> list[list]:
   ...
```

Then it can be called from other class or instance methods by referring to `cls` or `self`.

Reply:
Sounds good, I'll update those to be classmethods